Changelog.md

```diff
@@ -36,37 +37,45 @@
 - Allow restricting IP addresses for remote logins (#7072)
 
 ## [v2.4.9]
+
```

Comment:
Undo the other changes in this file (beyond the entry you added)

Reply:
Whoops, this seems to be the work of an autoformatter that I didn't catch because it doesn't render large diffs. Will revert this commit and fix, thanks!